app/policies/form526_policy.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Form526Policy = Struct.new(:user) do
```

Comment:
If you're not going to return `true` or `false` you probably shouldn't make this a `policy`, as it breaks the Pundit conventions. Is there any reason to not just call the `IdentificationVerifier` directly and forget the `_policy` part of this?

Reply:
Pundit is a Gem that expects certain conventions so it makes it pretty painful if we don't follow those

Reply:
Here's an example of how someone would expect to use a `policy`:
`before_action { authorize :appeals, :access? }`, if you were to do that with this policy it wouldn't work

Reply:
Yeah, this is a valid concern I was aware of the Pundit conventions but not sure how strictly they were enforced in this codebase (this is my first back end VA PR). The approach I took came as the [result of a suggestion](https://dsva.slack.com/archives/C053U7BUT27/p1690920021440909?thread_ts=1690830166.229329&cid=C053U7BUT27) from a team member with more experience in the codebase.

How strongly do you feel about re-writing this? I noticed you hadn't formally requested changes, is this a blocker for release?

Reply:
I personally feel pretty strongly about keeping convention in the `policies` folder. Rails is a framework built upon convention that doesn't technically need to be followed, but it basically is unusable if that convention isn't followed...

To show what I mean, if you were to use this policy like every single other one, you would do this:
`before_action { authorize :form526, :form526_required_identifier_presence }`
And it would differently than expected. This `authorize` action will always return `true` for this, since the `form526_required_identifier_presence` never returns `false`. 

I think `Users::Form526UserIdentificationVerifier` is already a service object with a great descriptive name, in my opinion it's a little weird to add what I'd consider tech debt right here when you absolutely don't need to